include/klee/Expr.h

```diff
@@ -1080,7 +1080,8 @@ class ConstantExpr : public Expr {
   }
 
   static ref<ConstantExpr> create(uint64_t v, Width w) {
-    assert(v == bits64::truncateToNBits(v, w) && "invalid constant");
+    if (w <= 64)
```

Comment:
The `NDEBUG` macro doesn't have anything to do with whether or not debugging info is enabled. It's to do with whether or not we are doing a build with assertions enabled. Both build systems allow for this to be controlled. The CMake build uses the `ENABLE_KLEE_ASSERTS` option to control this. The other build system can control this too but it's messy.